.github/actions/install-intel-graphics/action.yml

```
@@ -0,0 +1,41 @@
+name: install-intel-graphics
+description: Download and unpack Intel graphics drivers
+inputs:
+  GPU:
+    description: "Install for PVC or BMG gpu"
+    type: string
+  IGC:
+    description: "Use ROLLING or STAGING release"
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install Intel graphics drivers
+      if: inputs.IGC == 'ROLLING'
+      shell: bash
+      run: |
+        if [[ "${{ inputs.GPU }}" == "BMG" ]]; then
+          sudo add-apt-repository ppa:kobuk-team/intel-graphics
+          sudo apt-get install -y libze-intel-gpu1 libze1 intel-metrics-discovery intel-opencl-icd clinfo intel-gsc
+          sudo apt-get install -y libze-dev intel-ocloc
+        else 
+          sudo apt update
+          sudo apt install -y \
+            intel-opencl-icd libze-intel-gpu1 libze1 \
+            intel-media-va-driver-non-free libmfx-gen1 libvpl2 \
+            libegl-mesa0 libegl1-mesa-dev libgbm1 libgl1-mesa-dev libgl1-mesa-dri \
+            libglapi-mesa libgles2-mesa-dev libglx-mesa0 libigdgmm12 libxatracker2 mesa-va-drivers \
+            mesa-vdpau-drivers mesa-vulkan-drivers va-driver-all vainfo hwinfo clinfo 
+          sudo apt install -y \
+            libigc-dev intel-igc-cm libigdfcl-dev libigfxcmrt-dev libze-dev
+        fi
+    - name: Install Intel graphics drivers
+      if: inputs.IGC == 'STAGING'
+      shell: bash
+      run: |
+        if [[ "${{ inputs.GPU }}" == "BMG" ]]; then
+          sudo add-apt-repository ppa:kobuk-team/intel-graphics-staging
+          sudo apt-get install -y libze-intel-gpu1 libze1 intel-metrics-discovery intel-opencl-icd clinfo intel-gsc
+          sudo apt-get install -y libze-dev intel-ocloc
+        fi
```

There is no staging driver release for PVC. Does it make sense to return an error if someone tries to run this action with GPU = PVC and IGC = STAGING? 
```suggestion
        else
          exit 1
        fi
```